doc/introduction/importing_workflows.rst

```diff
@@ -0,0 +1,233 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing workflows
+===================
+
+PennyLane supports importing quantum circuits and operations that were
+constructed using another framework. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
+`OpenQASM <https://arxiv.org/abs/1707.03429>`_, and `Quil
+<https://docs.rigetti.com/qcs/guides/quil>`_.
+
+.. note::
+
+    To import a quantum circuit defined using a particular framework, you will need to install the
+    corresponding PennyLane plugin for that framework. More information about PennyLane plugins is
+    available on the `plugins <https://pennylane.ai/plugins.html>`_ page.
+
+Importing quantum circuits
+--------------------------
+
+Importing quantum circuits can help you take advantage of PennyLane's various optimization,
+visualization, and interoperability features for existing circuits. For example, you can take a
+`QuantumCircuit <https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit>`__ from
+Qiskit, import it into PennyLane, and then apply a `circuit-cutting transform
+<https://pennylane.ai/qml/demos/tutorial_quantum_circuit_cutting/>`_ to reduce the number of qubits
+required to implement the circuit. You could also :ref:`compile <intro_ref_compile_circuits>` the
+circuit using `Catalyst <https://docs.pennylane.ai/projects/catalyst/en/stable/index.html>`__ or
+differentiate and optimize the circuit using :ref:`quantum-specific optimizers <intro_ref_opt>`.
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
```

Comment:
For most other tables, I think we have the heading right above it, and the heading is something like "Functions for importing external workflows". This makes it easy to navigate to the reference list from the sidebar as soon as the page loads. 

Should we consider doing that here? We could for example move the table up just above "Importing quantum circuits" and give it its own main header, plus some brief line of text saying "The available functions for importing external workflows are:"

Reply:
I like that idea! Should we go ahead and do that, @trbromley?

Reply:
:thinking: I think that's maybe less of an established style in the quickstarts, e.g., I can see the following pages have a similar approach to what we have here already:
- Gradients and training
- Measurements
- Compiling circuits

I don't mind too much but I think it's nicer to read the quickstarts as more of a continuous story than broken up with extra section headings.

Reply:
Note that quickstarts are meant to be a more high level overview than the API docs:

> If the documentation is a map, the quickstarts are the introduction or 'legend'. The goal is to allow readers to navigate the core functionality of PennyLane (circuits, training, operators, measurements, manipulating circuits, etc.).
>
>This is helpful because
>
> - a lot of PennyLane functions are imported top level,
> - for new users familiar with quantum computation, it might not always be clear where corresponding PennyLane constructs live in the module structure, and
> - it allows us to surface *the* important functions/objects you should know, separate from everything else contained in the API section.
>
>When writing a quickstart, try to keep to the following structure:
>
> - **Summary tables**. Always provide an `autosummary` table of important functions/classes/objects
> - **Keep sections short.** Provide a minimal example if necessary, but avoid multiple code blocks/examples.
> - **Do not attempt to provide all usage details and explanation of an objects usage.** Remember, most users will be skimming through the quickstarts, using it as a reference to find their way. Instead, direct the reader to the docstring for more information, graphics, and usage details. In addition, this ensures that detailed usage/information is kept closer to the source code, where developers will remember to maintain it.
>
> And remember to keep in mind the  *Do's and don'ts* above! For example, we should be describing, not explaining.

Reply:
Sounds like we're sticking with no table headings for now.